packages/js-runtime/src/runtime/global/crypto.ts

```diff
@@ -40,11 +40,11 @@ interface CryptoKey {
     }
 
     async deriveBits(
```

Comment:
Could we add tests for this method? Probably in https://github.com/lagonapp/lagon/blob/main/crates/runtime/tests/crypto.rs

Reply:
Done, you can check it now.